Clinical-Genomics/cg
# fix bug with sorting novaseq sequenced dates

This PR adds a fix for sorting novaseq sequenced dates in the LIMS API

**How to test**:
Set test preconditions:

Log into Hasta
install branch on stage: bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-cg-stage lims-step-names-bug-fix
activate stage: us

TC1: get sequenced dates for microbial samples from LIMS
Execute Test:

prepare test data: delete sequenced_at dates in cg-stage by running the following command in your favorite sql tool: UPDATE `cg-stage`.microbial_sample SET sequenced_at = NULL
run cg transfer microbials --status sequenced
Expected Result:

Terminal messages Found new sequenced date for <microbial_sample_id>: <sequenced_at>, old value: None
Table `cg-stage`.microbial_sample updated with <sequenced_at> dates.
And most importantly, no TypeError in _most_recent_date(sequenced_dates) if novaseq sequenced dates are found :)

**Review:**
- [x] code approved by @ingkebil 
- [x] tests executed by @emiliaol
Thanks for filling in who performed the code review and the test!

This is a minor **version bump** because it's a simple bug fix
